bottles/frontend/ui/details-bottle.blp

```diff
@@ -269,6 +254,141 @@ template DetailsBottle : .AdwPreferencesPage {
 
   .AdwPreferencesGroup group_programs {
     title: _("Programs");
+
+    .AdwActionRow row_no_programs {
+      Box {
+        hexpand: true;
+
+        Label {
+          label: _("Use the Installers section or the \"Run executable\" button.");
+          wrap: false;
+
+          styles [
+            "dim-label",
+          ]
+        }
+      }
+    }
+    
+    .AdwPreferencesRow bottom_bar {
+      activatable: false;      
+      focusable: false;
+
+      Box {
+        Button add_shortcuts {
+          hexpand: true;
+          Box {
+            halign: center;
+            Image {
+              icon-name: "list-add-symbolic";
+            }
+            
+            Separator {
+              styles [
+                "spacer"
+              ]
+            }
+            
+            Label {
+              label: _("Add Shortcuts…");
+            }
+          }
+
+          styles [
+            "flat"
+          ]
+        }
+        
+        Separator {
+        }
+        
+        Button install_programs {
+          hexpand: true;
+          Box {
+            halign: center;
+            Image {
+              icon-name: "folder-download-symbolic";
+            }
+          
+            Separator {
+              styles [
+                "spacer"
+              ]
+            }
+
+            Label {
+              label: _("Install Programs…");
+            }
+          }
+          
+          styles [
+            "flat"
+          ]
+        }
+        
+        styles [
+          "linked"
+        ]
+      }
+      
+      styles [
+        "bottom-bar"
+      ]
+    }
+  }
+
+  .AdwPreferencesGroup group_options {
+    title: _("Options");
+      
+      .AdwActionRow row_preferences {
+        activatable: true;
+        title: _("Preferences");
+        subtitle: _("Configure bottle settings.");
+
+        Image {
+          icon-name: "go-next-symbolic";
+        }
+      }
+      
+      .AdwActionRow row_dependencies {
+        activatable: true;
+        title: _("Dependencies");
+        subtitle: _("Install dependencies for programs.");
+
+        Image {
+          icon-name: "go-next-symbolic";
+        }
+      }
+      
+      .AdwActionRow row_programs {
+        activatable: true;
+        title: _("Programs");
+        subtitle: _("Manage installed programs.");
+
+        Image {
+          icon-name: "go-next-symbolic";
+        }
+      }
```

Comment:
We'll have to remove this, as it received some complaints (see https://github.com/bottlesdevs/Bottles/issues/2126 and https://gitlab.gnome.org/Teams/Circle/-/issues/61).

I propose to have the Programs section in the details view the only way to manage programs. That way we won't have duplicated options and everything will be available.

Reply:
92200582ef83a388ac3f3565fb158cc8a478c5fa